frontend/src/components/object/ObjectPermission.vue

```diff
@@ -47,158 +51,166 @@ const updateObjectPermission = (value: boolean, userId: string, permCode: string
+            </div>
+          </div>
         </template>
-      </Column>
-    </DataTable>
-  </div>
+        <Column
+          field="fullName"
+          header="Name"
+        />
+        <Column
+          field="idpName"
+          header="Provider"
+        />
+        <Column
+          header="Read"
+          header-class="header-center"
+          body-class="content-center"
+        >
+          <template #body="{ data }">
+            <Checkbox
+              v-model="data.read"
+              input-id="read"
+              aria-label="read"
+              :binary="true"
+              :disabled="data.read"
+              @update:model-value="(value: boolean) => updateObjectPermission(value, data.userId, Permissions.READ)"
+            />
+          </template>
+        </Column>
+        <Column
+          header="Update"
+          header-class="header-center"
+          body-class="content-center"
+        >
+          <template #body="{ data }">
+            <Checkbox
+              v-model="data.update"
+              input-id="update"
+              aria-label="update"
+              :binary="true"
+              @update:model-value="(value: boolean) => updateObjectPermission(value, data.userId, Permissions.UPDATE)"
+            />
+          </template>
+        </Column>
+        <Column
+          header="Delete"
+          header-class="header-center"
+          body-class="content-center"
+        >
+          <template #body="{ data }">
+            <Checkbox
+              v-model="data.delete"
+              input-id="delete"
+              aria-label="delete"
+              :binary="true"
+              @update:model-value="(value: boolean) => updateObjectPermission(value, data.userId, Permissions.DELETE)"
+            />
+          </template>
+        </Column>
+        <Column
+          header="Manage"
+          header-class="header-center"
+          body-class="content-center"
+        >
+          <template #body="{ data }">
+            <Checkbox
+              v-model="data.manage"
+              input-id="manage"
+              aria-label="manage"
+              :binary="true"
+              @update:model-value="(value: boolean) => updateObjectPermission(value, data.userId, Permissions.MANAGE)"
+            />
+          </template>
+        </Column>
+        <Column header="Remove">
+          <template #body="{ data }">
+            <Button
+              class="p-button-lg p-button-text"
+              severity="danger"
+              aria-label="remove"
+              @click="removeObjectUser(data.userId)"
+              @keyup.enter="removeObjectUser(data.userId)"
+            >
+              <font-awesome-icon icon="fa-solid fa-user-xmark" />
+            </Button>
+          </template>
+        </Column>
+      </DataTable>
+    </TabPanel>
+    <TabPanel header="Bulk add/remove">
+      <BulkPermission
+        resource-type="object"
+        :resource="object"
```

Comment:
despite the diff looking big, the only change to this file is putting the content into two tabs, and passing the `object` to the BulkPermission component.